scala/scala-impl/src/org/jetbrains/sbt/project/MillProjectImportProvider.scala

```diff
@@ -0,0 +1,23 @@
+package org.jetbrains.sbt.project
+
+import com.intellij.openapi.vfs.VirtualFile
+import scala.io.Source
+
+object MillProjectImportProvider {
+  private val buildFile = "build.sc"
+
+  def canImport(vFile: VirtualFile): Boolean = vFile match {
+    case null => false
+    case directory if directory.isDirectory =>
+      val buildWithBspPlugin = directory.getChildren.exists { vBuildScript =>
+        vBuildScript.getName == "build.sc" &&
+          Source.fromInputStream(vBuildScript.getInputStream)
+            .getLines()
+            .contains("import $ivy.`com.lihaoyi::mill-contrib-bsp:$MILL_VERSION`")
```

Comment:
Would it make sense to not make this detection dependent on this specific line to be in the build.sc? Are there situations where one would want to put a specific mill version instead?

Reply:
I can inject the plugin, that would solve the issue.

Reply:
Great, hope we can still get https://github.com/lihaoyi/mill/issues/899 sooner than later

Reply:
Injecting it sounds not as easy as adding a file with the plugin and it feels like a waist of time to implement this since eventually we will get to https://github.com/lihaoyi/mill/issues/899.